tests/acceptance/curriculum-inventory/leadership-test.js

```diff
@@ -13,44 +13,48 @@ module('Acceptance | curriculum inventory leadership', function (hooks) {
       school: this.school,
       administeredSchools: [this.school],
     });
-
     const program = this.server.create('program', { school: this.school });
     const users = this.server.createList('user', 4);
-    this.server.create('curriculum-inventory-report', {
+    this.report = this.server.create('curriculum-inventory-report', {
       program,
       administrators: [users[2], users[3]],
     });
   });
 
   test('collapsed leadership', async function (assert) {
-    assert.expect(6);
-    await page.visit({ reportId: 1 });
-
+    const reportModel = await this.owner
+      .lookup('service:store')
+      .find('curriculumInventoryReport', this.report.id);
+    await page.visit({ reportId: reportModel.id });
     assert.equal(page.details.leadershipCollapsed.title, 'Curriculum Inventory Report Leadership');
     assert.equal(page.details.leadershipCollapsed.headers.length, 1);
     assert.equal(page.details.leadershipCollapsed.headers[0].title, 'Summary');
-
     assert.equal(page.details.leadershipCollapsed.summary.length, 1);
     assert.equal(page.details.leadershipCollapsed.summary[0].name, 'Administrators');
     assert.equal(page.details.leadershipCollapsed.summary[0].value, 'There are 2 administrators');
   });
 
   test('list leadership', async function (assert) {
-    assert.expect(4);
-    await page.visit({ reportId: 1, leadershipDetails: true });
-
-    assert.equal(page.details.leadershipExpanded.title, 'Curriculum Inventory Report Leadership');
+    const reportModel = await this.owner
+      .lookup('service:store')
+      .find('curriculumInventoryReport', this.report.id);
+    await page.visit({ reportId: reportModel.id, leadershipDetails: true });
+    assert.equal(
+      page.details.leadershipExpanded.collapse.text,
+      'Curriculum Inventory Report Leadership'
+    );
     const { administrators } = page.details.leadershipExpanded.leadershipList;
-
     assert.equal(administrators.length, 2);
     assert.equal(administrators[0].text, '3 guy M. Mc3son');
     assert.equal(administrators[1].text, '4 guy M. Mc4son');
   });
 
   test('search administrators', async function (assert) {
-    assert.expect(11);
-    await page.visit({ reportId: 1, leadershipDetails: true });
-    await page.details.leadershipExpanded.manage();
+    const reportModel = await this.owner
```

Comment:
I don't understand why you're pulling the report here only to use the id that you used to pull the report?

Reply:
yea that's wrong. fixing it...

Reply:
handled.

Reply:
Exists in a few other files as well. I guess I don't mind it, but it seems like the magic number is just being used to fetch a different thing from the DB whereas before it was being used directly. Having reportModel isn't a bad change though and is readable so I'll take it. Want me to mark the other files this is also needed in?

Reply:
i found two more acceptance tests that use the magic report id. fixing them up now..